index.html

```diff
@@ -157,43 +157,12 @@ <h1>Introduction</h1>
         <p>
             Binding Templates consist of multiple specifications, referred to as a <a>subspecification</a>
              in this document, that enable an application client (a WoT Consumer) to interact, using WoT Thing 
-             Description[[WOT-THING-DESCRIPTION]] (TD), with Things that exhibit diverse protocols, payload formats and their usage.
-            These subspecifications are categorized into three:
-        <ul>
-            <li> <a href="#protocol-bindings"></a>: Application layer protocols (e.g., HTTP[[?RFC7231]], CoAP[[?RFC7252]],
-                MQTT[[?MQTT]], etc.) whose different message types are mapped to the WoT Thing
-                Description[[WOT-THING-DESCRIPTION]] forms via reusable vocabulary and extensions.</li>
-            <li> <a href="#payload-bindings-intro"></a>: Different payload formats and media types [[IANA-MEDIA-TYPES]] which
-                can be represented in a TD via Data Schemas or forms.</li>
-            <li> <a href="#platforms-intro"></a>: Platforms and frameworks who combine the use of protocols and payloads
-                in a certain way, which can be represented via entire Thing Models described by the [[WOT-THING-DESCRIPTION]]
-                or examples of TDs.</li>
-        </ul>
+             Description[[WOT-THING-DESCRIPTION]] (TD), with Things that exhibit diverse protocols, payload formats and a
+             combination of these in platforms and frameworks.
+             The mechanism that allows this is called the Binding Mechanism, without which TDs could not build Hypermedia Controls
```

Comment:
*allow this* what?

Reply:
I have added some text but it is a bit difficult to express this in a short/elegant way